.devcontainer/docker-compose.yml

```diff
@@ -17,6 +17,7 @@ services:
       FLASK_DEBUG: "True"
       PORT: 8000
       DATABASE_URI: postgresql://postgres:postgres@postgres:5432/postgres
+      WAIT_SECONDS: 5
```

Comment:
While troubleshooting, I copied/pasted the contents of docker-compose.yml in the BDD class lab into our docker-compose.yml file. It doesn't look like anything broke?